lib/Files.js

```diff
@@ -39,6 +39,12 @@ function throwNotImplemented (funcName) {
   logAndThrow(new codes.ERROR_NOT_IMPLEMENTED({ messageValues: [funcName] }))
 }
 
+const _permissions = {
```

Comment:
Here you can define a static constant instead: `Files.Permissions = {}` that way it is exposed to the user too